src/sonic-config-engine/minigraph.py

```diff
@@ -47,6 +47,26 @@ def default(self, obj):
             return str(obj)
         return json.JSONEncoder.default(self, obj)
 
+def parse_device(device):
+    lo_prefix = None
```

Comment:
Let's keep it None as returning an empty string might make template debug mroe difficult than None.

---
In reply to: [125150571](https://github.com/Azure/sonic-buildimage/pull/775#discussion_r125150571) [](ancestors = 125150571)